src/lib.zig

```diff
@@ -5,6 +5,10 @@ pub const c = @cImport({
     @cInclude("TracyC.h");
 });
 
+pub fn frame() callconv(.Inline) void {
```

Comment:
I noticed from your diff, there's no `frameEnd` in this change and was curious where that went?

Reply:
Tracy has three functions for frame marking [in `TracyC.h`](https://github.com/wolfpld/tracy/blob/master/TracyC.h#L190-L192):

```
TRACY_API void ___tracy_emit_frame_mark( const char* name );
TRACY_API void ___tracy_emit_frame_mark_start( const char* name );
TRACY_API void ___tracy_emit_frame_mark_end( const char* name );
```

I first had `frameStart` and `frameEnd`, but then I tried out `___tracy_emit_frame_mark` and it seems that tracy autodetects frame start + end when this function is used.  I thought it might be a bit simpler to just call `tracy.frame` at the start of each frame and let `tracy` work it out.

But we could also make a little `Frame` struct like there is for `Ctx` and implement the same API like there is for `tracy.trace`?  I.e. it would look like this:

```zig
while (!quit) {
    const frame = tracy.frame();
    defer frame.end();
    
    // rest of the loop
}
```

This looks a bit more flexible, so maybe it's better, what do you think?

Reply:
are you familiar with how its able to detect the bound of the last frame without a file `frame_mark` call?

Reply:
I am not.  My guess is that `___tracy_emit_frame_mark` will just count the time from the last call to the current one the frame duration.

I'll prototype the `Frame` struct as I described above, because that seems a more reliable way to handle this.

Reply:
Did the refactoring, I think it looks quite nice now.